src/Compilers/CSharp/Test/Symbol/Symbols/PartialPropertiesTests.cs

```diff
@@ -955,33 +1013,199 @@ partial class C
         }
 
         [Fact]
-        public void ModifierDifference_01()
```

Comment:
nit: The diff becomes pretty confused at this point. In the future, consider keeping original test names and adding new tests at the end to help the review